src/status_im/contexts/onboarding/intro/view.cljs

```diff
@@ -44,9 +44,13 @@
        :style  style/plain-text}
       (i18n/label :t/by-continuing-you-accept)]
      [quo/text
-      {:on-press #(debounce/throttle-and-dispatch [:open-modal :privacy-policy] 1000)
-       :size     :paragraph-2
-       :weight   :regular
-       :style    style/highlighted-text}
+      {:on-press
+       #(debounce/throttle-and-dispatch
+         [:browser.ui/open-url
+          "https://raw.githubusercontent.com/status-im/status-desktop/master/ui/imports/assets/docs/terms-of-use.mdwn"]
```

Comment:
should we put it in a constant? 🤔

Reply:
i don't think so, it's a temporary solution, so as less as possible changes should be done